cmd/commands/restore.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/restore"
+	"github.com/spf13/cobra"
+)
+
+// RestoreCmd represents the restore commands
+var RestoreCmd = &cobra.Command{
+	Use:   "restore-deleted",
+	Short: "Reads the required arg CRD and optional CR name to restore. Ex: restore cephcluster <my-cluster>",
```

Comment:
Let's describe what the command does. The arguments are shown by the help command anyway, right? No need to describe them here?
```suggestion
	Short: "Restores a CR that was accidentally deleted and is still in terminating state.",
```

Reply:
correct

Reply:
actually, what we describe in `Short:` is printed in help command